lib/osf-components/addon/components/registries/schema-block-renderer/multi-select-input/template.hbs

```diff
@@ -0,0 +1,23 @@
+<FormControls
+    @changeset={{@changeset}}
+    @disabled={{@disabled}}
+    @shouldShowMessages={{@shouldShowMessages}}
+    as |form|
+>
+    <form.checkboxes
+        data-test-multi-select-input
+        @id={{@schemaBlock.registrationResponseKey}}
```

Comment:
Is `id` necessary?

Reply:
It's used to create the ids for each checkbox:
https://github.com/CenterForOpenScience/ember-osf-web/blob/2b851999b94b3e64ed0306ab336c5cc60e9ff20e/lib/osf-components/addon/components/validated-input/checkboxes/template.hbs#L8-L23

Reply:
But also, it *might* not be that important to have these be meaningful as long as they are unique. vOv